src/components/DropzoneDialogBase.js

```diff
@@ -8,96 +8,66 @@ import * as React from 'react';
 
 import DropzoneAreaBase from './DropzoneAreaBase';
 
-// Split props related to DropzoneDialog from DropzoneArea ones
-function splitDropzoneDialogProps(allProps) {
-    const {
-        cancelButtonText,
-        dialogProps,
-        dialogTitle,
-        fullWidth,
-        maxWidth,
-        onClose,
-        onSave,
-        open,
-        submitButtonText,
-        ...dropzoneAreaProps
-    } = allProps;
-
-    return [
-        {
-            cancelButtonText,
-            dialogProps,
-            dialogTitle,
-            fullWidth,
-            maxWidth,
-            onClose,
-            onSave,
-            open,
-            submitButtonText,
-        },
-        dropzoneAreaProps,
-    ];
-}
-
 /**
  * This component provides the DropzoneArea inside of a Material-UI Dialog.
  *
  * It supports all the Props and Methods from `DropzoneAreaBase`.
  */
-class DropzoneDialogBase extends React.PureComponent {
-    render() {
-        const [dropzoneDialogProps, dropzoneAreaProps] = splitDropzoneDialogProps(this.props);
-        const {
-            cancelButtonText,
-            dialogProps,
-            dialogTitle,
-            fullWidth,
-            maxWidth,
-            onClose,
-            onSave,
-            open,
-            submitButtonText,
-        } = dropzoneDialogProps;
+const DropzoneDialogBase = ({
+    cancelButtonText,
+    dialogProps,
+    dialogTitle,
+    fullWidth,
+    maxWidth,
+    onClose,
+    onSave,
+    open,
+    submitButtonText,
+    ...other
```

Comment:
Instead of `other` I'd keep `dropzoneAreaProps`

Reply:
this change has been pushed, I originally thought this had to be `other` however I have just learnt that this i you can actually choose your own variable name for the rest of the props,